packages/account/src/account.ts

```diff
@@ -665,7 +669,8 @@ export class Account {
     await this.tracker.savePresignedConfiguration({
       wallet: this.address,
       nextConfig: config,
-      signature
+      signature,
+      validateBehavior: 'ignore'
```

Comment:
Why are we using `ignore` here? what's the difference?
I understand next line should caught it anyway, but I still wonder.

Reply:
This is to prevent the issue with validating EIP-1271 signatures mentioned in the other comments.